internal/config/config_type.go

```diff
@@ -271,7 +271,34 @@ func (c *fileConfig) GetWithSource(hostname, key string) (string, string, error)
 	return value, defaultSource, nil
 }
 
+func validConfigEntries(key string) []string {
```

Comment:
Nit: this function could be called `validConfigValues` to describe better its return value